### PR TITLE
Keepercore remote CLI

### PR DESCRIPTION
**Pull request by lloesche**
MVP of a very simple CLI that calls keepercore's `/cli/*` endpoints

### CODE REVIEW

File: cloudkeeper-cli/cloudkeeper_cli/__init__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+class VersionInfo(NamedTuple):
+    major: int
+    minor: int
+    micro: int
+    releaselevel: Literal["alpha", "beta", "candidate", "final"]
+    serial: int
+
+
+version_info: VersionInfo = VersionInfo(
+    major=2, minor=0, micro=0, releaselevel="alpha", serial=0
```
Comment (aquamatthias) [inline cloudkeeper-cli/cloudkeeper_cli/__init__.py:27]: 2?

**Review by aquamatthias — APPROVED** [review]
nice - thanks!

File: cloudkeeper-cli/cloudkeeper_cli/__init__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+A commandline interpreter to interact with cloudkeeper.
+:copyright: (c) 2021 Some Engineering Inc.
+:license: Apache 2.0, see LICENSE for more details.
+"""
+
+__title__ = "cloudkeeper-cli"
+__author__ = "Some Engineering Inc."
+__license__ = "Apache 2.0"
+__copyright__ = "Copyright 2021 Some Engineering Inc."
+__version__ = "0.0.1a"
```
Comment (aquamatthias) [inline cloudkeeper-cli/cloudkeeper_cli/__init__.py:13]: and the version correlates?

File: cloudkeeper-cli/cloudkeeper_cli/__init__.py (hunk 1 above)
Comment (lloesche) [reply cloudkeeper-cli/cloudkeeper_cli/__init__.py]: Copy'paste typo

**Pull request merged**